AMP_CACHE_GUIDELINES.md

```diff
@@ -6,35 +6,35 @@ Provide guidelines to external parties for implementing their own AMP Caches.
 
 ## Guidelines: Core
 
-1. An AMP Cache only serves [valid AMP documents](https://github.com/ampproject/amphtml/blob/master/spec/amp-html-format.md).
+1. An AMP Cache only serves [valid AMP input documents](https://github.com/ampproject/amphtml/blob/master/spec/amp-html-format.md).
 
 2. It participates in the AMP Project's [validator release cycle](https://github.com/ampproject/amphtml/tree/master/validator).
 
 3. It participates in the AMP JS library release cycle and makes every effort to serve the latest version. It does not allow sites to perform version locking.
 
 4. Pledges to maintain URL space forever (even beyond the lifetime of the cache itself): 
 
-    1. This can be achieved by donating the URL space to a trustworthy third party entity such as [archive.org](http://archive.org).
+    4.1. This can be achieved by donating the URL space to a trustworthy third party entity such as [archive.org](http://archive.org).
```

Comment:
This won't translate into proper Markdown. Just using `   1.` with indentation will.